packages/sites/src/use-sites-list-filtering.tsx

```diff
@@ -9,9 +9,9 @@ export interface SitesFilterOptions {
 	includeA8CSites?: boolean;
 }
 
-type SiteForFiltering = Pick< MinimumSite, 'URL' | 'name' | 'slug' | 'title' | 'site_owner' >;
+type SiteForFiltering = Pick< MinimumSite, 'URL' | 'name' | 'slug' | 'title' | 'is_a8c' >;
 
-const isA8CSite = ( site: SiteForFiltering ) => site.site_owner === 26957695;
+const isA8CSite = ( site: SiteForFiltering ) => site.is_a8c;
```

Comment:
I no longer see much value in this function. Perhaps we can access the property directly on line 28. Not a strong opinion though.
Or we could use Ella's function if dependencies make sense.

Reply:
Thanks, Antonio! That's a good catch! I have made an update in https://github.com/Automattic/wp-calypso/pull/102970/commits/bda1377a008381553f47ae9fce58d728092e5cab.